test/logic/QuotientTest.java

```diff
@@ -36,7 +36,7 @@ public void quotientConstructorShouldAddANewClockWithNameQuo_new() {
         assertEquals(quotient.getClocks().size(), 1);
         assertTrue(contains_clock_quo_new);
     }
-
+    
```

Comment:
Remove white space characters so it becomes an empty line.

Reply:
Nice catch.